config/hardware/axis/XY/TMC/xy_TMC2209_V0specific.cfg

```diff
@@ -1,12 +1,11 @@
-[stepper_x]
-rotation_distance: 40
-microsteps: 64
-full_steps_per_rotation: 200
+# XY TMC2209 definition
 
+# Including first the default XY TMC2209 file
+[include xy_TMC2209.cfg]
+
+# Then applying overrides for specific V0 parameters from LDO to reduce VFAs
```

Comment:
oh, the formbot v0 kits are not delivered with LDO. shouldn't be some central info that this settings are LDO specific?
I can't unoverride settings for a formbot kit

Reply:
Yes, using a standard V0 kit should use the standard V0 file. This specifics are used only for the problematic VFA generated by first generation of LDO motors (kit or not kit as you can buy them also by yourself). There were replaced and this should not be needed by newer generation of motors.

I can add a comment like : "if you use LDO motors on your V0 and get a lot of VFAs, try to use the V0 specific file instead"